src/main/resources/db/changelog/changes/06-assign-roles.yaml

```
@@ -0,0 +1,90 @@
+databaseChangeLog:
+  - changeSet:
+      id: update-users-table
+      author: Yaroslav
+      changes:
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: ADMIN
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: USER
+
+  - changeSet:
+      id: insert-user1
+      author: Yaroslav
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  value: 1
+              - column:
+                  name: email
+                  value: gates@email.com
+              - column:
+                  name: password
+                  value: $2a$10$AduObuNZCl3w.z39uaTzSur114EA2y4RsmgoI3xEZySbEcVGw8PpW
```

```suggestion
                  value: $2a$10$AduObuNZCl3w.z39uaTzSur114EA2y4RsmgoI3xEZySbEcVGw8PpW #1234
```
add comments to the hashed passwords, otherwise what's the point, only you know them and can use those dummy users